option.go

```diff
@@ -58,6 +59,13 @@ func WithHotReload() Option {
 	}
 }
 
+// WithBeginAtTop makes the selection begin at the top of the list instead of the bottom
+func WithBeginAtTop() Option {
```

Comment:
I think we should avoid adding an option which enables a specific layout. Instead, it's more extensible to add an option that can specify a layout from a list of layouts. For example, [`WithMode`](https://pkg.go.dev/github.com/ktr0731/go-fuzzyfinder#WithMode) option can specify one of the provided modes.

What do you think about the following design? `CursorPositionBottom` is the default position (the same as the current implementation).

```suggestion
type cursorPosition int

const (
  CursorPositionBottom cursorPosition = iota
  CursorPositionTop
)

func WithCursorPosition(position CursorPosition) Option {
```